wrangler-service/src/main/java/io/cdap/wrangler/service/directive/WorkspaceHandler.java

```diff
@@ -99,10 +99,16 @@ public class WorkspaceHandler extends AbstractDirectiveHandler {
   private static final Gson GSON =
     new GsonBuilder().registerTypeAdapter(Schema.class, new SchemaTypeAdapter()).create();
   private static final Pattern PRAGMA_PATTERN = Pattern.compile("^\\s*#pragma\\s+load-directives\\s+");
+  private static final String UPLOAD_COUNT = "upload.file.count";
+  private static final String UPLOAD_TYPE = "upload";
```

Comment:
nit: Does it makes sense to rename it as `CONNECTION_TYPE` instead of `UPLOAD_TYPE`?

Reply:
Done